src/VkBreadcrumb.php

```diff
@@ -162,7 +162,7 @@ public static function get_array() {
 
 		if ( is_date() ) {
 			$breadcrumb_array[] = array(
-				'name'  => get_the_archive_title(),
+				'name'  => date( 'Y' ),
```

Comment:
@shimotmk おや？ここは 

* 表示対象が年別とは限らない（月別か日別もある）
* get_the_archive_title() の中のフックを使ったりする（している）ので、使えなくなるのは困る

という理由で、get_the_archive_title() じゃないと都合が悪いです ＞＜

```suggestion
				'name'  => get_the_archive_title(),
```